Web/scripts/fluid_properties.IF97Conformance.py

```
@@ -293,60 +293,211 @@ def run_conformance(backend):
+    if Tc is not None:
+        T_tri = max(273.16, ref.Ttriple() if hasattr(ref, 'Ttriple') else 273.16)
+        for i in range(200):
+            T = T_tri + (Tc - T_tri) * (i / 199.0)
+            try:
+                p_sat = CP.PropsSI('P', 'T', T, 'Q', 0, 'IF97::Water')
+                h_L = CP.PropsSI('H', 'T', T, 'Q', 0, 'IF97::Water')
+                h_V = CP.PropsSI('H', 'T', T, 'Q', 1, 'IF97::Water')
+                sat_h.append((h_L, h_V))
+                sat_p.append(p_sat)
+            except Exception:
+                pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)